django/db/backends/postgresql/introspection.py

```diff
@@ -145,7 +142,6 @@ def get_indexes(self, cursor, table_name):
                 indexes[row[0]]['primary_key'] = True
             if row[2]:
                 indexes[row[0]]['unique'] = True
-            indexes[row[0]]['type'] = row[4]
         return indexes
 
     def get_constraints(self, cursor, table_name):
```

Comment:
[This docstring](https://github.com/akki/django/blob/ccadc4b54e859d40db25638a470deb1333c7ac4c/django/db/backends/postgresql/introspection.py#L175) only says that this method returns constraints or keys, which I think is still true (and complete).
The details of the returned dictionaries is written in [base.introspection](https://github.com/akki/django/blob/ccadc4b54e859d40db25638a470deb1333c7ac4c/django/db/backends/base/introspection.py#L175) which I have updated.

What updations do you suggest in this postgresql specific method's docstring?